numpy/distutils/ccompiler_opt.py

```diff
@@ -152,6 +152,18 @@ class _Config:
             By default(None), treated as True if the feature contains at
             least one applicable flag. see `feature_can_autovec()`
 
+        "extra_checks": str or list, optional
+            Extra test cases names for the CPU feature that need to be tested
+            against the compiler.
+
+            Each test case name must has a C file located at 'conf_check_path'
+            contains at least one intrinsic or function related to the test case
+            named extra_xxxx.c where xxxx is the case name in lower case.
+
+            If the compiler able to successfully compile the C file then `CCompilerOpt`
+            will add C #definition for it into the main dispatch header, e.g.
+            #define {conf_c_prefix}_XXXX where XXXX is the case name in upper case.
```

Comment:
Grammar is slightly off here:
```suggestion
        "extra_checks": str or list, optional
            Extra test case names for the CPU feature that need to be tested
            against the compiler.

            Each test case must have a C file named ``extra_xxxx.c``, where
            ``xxxx`` is the case name in lower case, under 'conf_check_path'.
            It should contain at least one intrinsic or function related to the test case.

            If the compiler able to successfully compile the C file then `CCompilerOpt`
            will add C #definition for it into the main dispatch header, e.g.
            ```#define {conf_c_prefix}_XXXX`` where ``XXXX`` is the case name in upper case.
```